packages/@react-aria/select/src/useSelect.ts

```diff
@@ -50,9 +59,11 @@ interface SelectAria {
  * @param props - Props for the select.
  * @param state - State for the select, as returned by `useListState`.
  */
-export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>, ref: RefObject<HTMLElement>): SelectAria {
```

Comment:
yes - I used the same logic you find in useComboBox. There was no public onBlur/onFocus/onFocusChange before because we did not inherit from `FocusableProps` - in the ticket it was said that I should give that route a try.

As I outlined in the ticket my sole use case today is that I can autoFocus the Picker on render - so adding `FocusableProps` is not strictly needed for my UC

Reply:
Ah, thanks for the pointer to useComboBox, that makes more sense now. Sorry I missed that